src/main/java/org/jenkinsci/test/acceptance/po/Jenkins.java

```diff
@@ -164,21 +166,17 @@ public PluginManager getPluginManager() {
         return new PluginManager(this);
     }
 
-    /** 
-     * Some tests require they restart Jenkins - but depending on how the SUT is launched this is not always possible
-     * so tests that require this should wrap this call in an {@link org.junit.Assume#assumeTrue(String, boolean)}
-     * @return true if the Jenkins under test can restart itself.
-     */
     public boolean canRestart() {
-        visit("restart");
-        return getElement(by.button("Yes")) != null;
+        return true;
```

Comment:
Unclear from 7c40dfb37529ee1d9a65ff48af9c46712b238706 why this is being done here.

Reply:
https://github.com/jenkinsci/acceptance-test-harness/pull/711#issuecomment-1080647705

Reply:
So IIUC this now uses https://github.com/jenkinsci/acceptance-test-harness/blob/a9027d40be247fbc51f4ea4af7e4bd0bd7f7e6e8/src/main/java/org/jenkinsci/test/acceptance/controller/JenkinsController.java#L148-L156 which ignores the `hudson.Lifecycle` (and thus the crazy JNI hacks) and just shuts down and restarts the controller using the external system? Like https://github.com/jenkinsci/jenkins/pull/5899#issuecomment-999660881

Can we deprecate `canRestart` and delete existing calls?